setupTests-integration.js

```diff
@@ -21,11 +20,8 @@ beforeAll(async () => {
   const testLogger = new TxLogger(testName);
   testLogger.init();
   loggers.test = testLogger;
-
-  await TestUtils.startWallet(WALLET_CONSTANTS.genesis);
 });
 
 // This function will run after each test file is executed
 afterAll(async () => {
```

Comment:
Remove `afterAll`?

Reply:
I kept it for easier reference, should anyone need to use it later.
But since there already is the `setupTests.js` file with this same reference code, it's better to remove it indeed. :+1: